.github/workflows/tests-docker.yml

```diff
@@ -63,6 +63,16 @@ jobs:
         python3 wis2box-ctl.py execute wis2box auth add-token -p wis2downloader github123 -y
         echo '{"topic": "origin/a/wis2/+/data/core/#"}' > test.json
         curl -X POST http://localhost/wis2downloader/subscriptions -H "Content-Type: application/json" -H "Authorization: Bearer github123" -d @test.json
+    - name: add Brazil wrongly encoded station data and check for error message
+      env:
+        STATION_METADATA: /data/wis2box/metadata/station/brazil.csv
+      run: |
```

Comment:
We should not be encoding test logic in CI.  This test should be in put forth in `tests/integration/test_workflow.py` instead.

Reply:
@tomkralidis, I disagree with your comment you asked Alberto to write the test in CI in your previous request and this test is based on command-line logic to demonstrate that the user receives a clear error when ingesting stations from the command-line, we should **not** put this in `tests/integration/test_workflow.py`